### PR TITLE
Withdrawal RPC

### PR DESCRIPTION
**Pull request by Krigpl**
For withdrawing GNT/ETH.

### CODE REVIEW

File: golem/transactions/ethereum/ethereumtransactionsystem.py [hunk 1]
```diff
@@ -76,4 +77,58 @@ def get_balance(self):
...
+            if amount > eth:
+                raise ValueError('Not enough ETH available')
+            log.info(
+                "Withdrawing %f ETH to %s",
+                amount / denoms.ether,
+                destination,
+            )
+            return [self._sci.transfer_eth(destination, amount)]
+
+        if currency == 'GNT':
```
Comment (mfranciszkiewicz) [inline golem/transactions/ethereum/ethereumtransactionsystem.py:99]: Is it safe to assume that we have enough `ETH` for the transfer / conversion?

File: golem/client.py [hunk 2]
```diff
@@ -849,6 +849,19 @@ def get_payments_list(self):
     def get_incomes_list(self):
         return self.transaction_system.get_incoming_payments()
 
+    def withdraw(
+            self,
+            amount: Union[str, int],
+            destination: str,
+            currency: str) -> List[str]:
+        if not self.is_mainnet():
```
Comment (mfranciszkiewicz) [inline golem/client.py]: Could you change this call to `if not self.mainnet`? `is_mainnet()` has been moved to `Node` in `rpc_pass` branch

File: golem/transactions/ethereum/ethereumtransactionsystem.py (hunk 1 above)
Comment (Krigpl) [reply golem/transactions/ethereum/ethereumtransactionsystem.py:99]: No :)

File: golem/client.py (hunk 2 above)
Comment (Krigpl) [reply golem/client.py]: Done

**Review by mfranciszkiewicz — APPROVED** [review]

**Pull request merged**